guides/common/modules/proc_installing-flatpak-applications.adoc

```diff
@@ -0,0 +1,41 @@
+[id="installing-flatpak-applications_{context}"]
+= Installing Flatpak applications
+
+This procedure installs selected applications from the enabled Flatpak remotes on the command line.
+
+.Prerequisites
+* Flatpak is installed.
+* The Red Hat Flatpak remote is enabled.
+
+.Procedure
+. Run the following command on the managed host to add the {Project} as a Flatpak remote:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak --user remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://satellite.example.com/
```

Comment:
Lets remove --user from here and add a note for context if needed. The --user flag sets up remote and also installs apps in user context on the client. System-wide installs are the default and the REX templates also follow that.